free-1.md

```diff
@@ -4,47 +4,143 @@
 
+This project idea excites me because I've written a lot of HTML, CSS, and JS in
+the past year. I've explored many tools and framework that make things easier.
+A good example is React. With React, I used the JSX syntax, which I thought was
+a good abstraction that made code shorter to write than if I had written the
+same things in plain javascript. However, it also didn't lose any meaning - once
+I got the translation/abstraction, the code was cleaner to read, shorter, and
+easier to debug. However, JSX is mostly accessible in React. Also, it would be
+a little different (in my mind) if something similar could be used to write
+html, css, and javascript without the React framework (as JSX is very custom for
+React and things related to how React is used), so more work would be needed and
+it would not just be a simple port from JSX.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Making webdev easier and nicer.
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
-
+_How might the user interact with the language? What does programming look
+like? Why is this the right way to interact with the problem domain?_
+
+The user will interact with the language through coding. Programming would look
+like the user is writing a dialect/hybrid of HTML, CSS, and JavaScript, but with
+nicer syntax. This is the right way to interact with the problem domain because
+people will still need to write code, but a DSL can make that simpler and nicer.
+Since they are already writing code, they are not the people that use Wix or
+Squarespace to create websites through drag-and-drop, but they would not be
+averse to coding. However, they would be more efficient with smaller syntax and
+needing to write less code.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+It should be able to compile and translate what the user has written in this
+shorter syntax into proper HTML, CSS, and Javascript.
+
+The possible errors that might occur are bad syntax from the user. This can be
+communicated to the user through errors during the translation, or through
+modifying the files (like what git does when there is a conflict).
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+Writing html/css/javascript should be easy to do in this language. Perhaps
+deeper manipulations that should be done in pure javascript should be possible,
+but difficult (and syntax would potentially be ugly, defeating its purpose).
+Doing backend, none front-end work should be impossible or very difficult. This
+means interacting with databases and servers should be impossible or very
+difficult.
```

Comment:
I think this description of what's easy and difficult makes a lot of sense given the goals of the language. If it's going to be difficult to do those "deeper manipulations", do you imagine people using your language together with plain JS in the same project? How would these languages interface with each other?